content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -1,99 +1,190 @@
+
+.. image:: down_payment/down-payment-line-item.png
    :align: center
-   :alt: What is a basic sales flow using down payments on Odoo Sales (3)?
+   :alt: How the down payment is mentioned on the sales order in Odoo Sales.
 
-This flow is also possible with a down payment taking into account a fixed amount.
+To confirm and register the first payment of the down payment, click the :guilabel:`Invoices` smart
+button to return to the invoice. On the :guilabel:`Customer Invoice` page, click
+:guilabel:`Confirm`, then :guilabel:`Register Payment` to open a :guilabel:`Register Payment`
+pop-up form.
+
+.. image:: down_payment/register-payment-popup.png
+   :align: center
+   :alt: How the register payment pop up looks in Odoo Sales.
+
+On this form, select the appropriate :guilabel:`Journal` and confirm that the down payment amount
+is correct in the :guilabel:`Amount` field. Then, make any other modifications to the other fields,
+if necessary, before clicking :guilabel:`Create Payment`.
+
+Odoo returns to the invoice, which now has a green `Paid` banner in the upper-right corner.
+
+.. image:: down_payment/paid-banner-invoice-1.png
+   :align: center
+   :alt: How the initial down payment invoice has a green paid banner in Odoo Sales.
+
+Now, when the customer wants to pay the remaining amount of the order, another invoice must be
+created. To do that, return to the sales order, and click :guilabel:`Create Invoice`.
+
+On the :guilabel:`Create invoices` pop-up, there is now an option to :guilabel:`Deduct down
+payments`, which is selected by default.
+
+.. image:: down_payment/deduct-down-payments-option.png
+   :align: center
+   :alt: The deduct down payment option on the create invoices pop up in Odoo Sales.
+
+If the remaining amount is ready to be paid, the :guilabel:`Regular Invoice` option should be
+selected, and the :guilabel:`Deduct down payments` option should *stay* selected. Then, click
+:guilabel:`Create and View Invoice`.
+
+Doing so reveals a separate :guilabel:`Invoices` page, listing all the invoices for that specific
+sales order. Each invoice line item displays all the necessary information related to each invoice,
+along with the current :guilabel:`Payment Status`.
+
+.. image:: down_payment/down-payment-invoices-page.png
+   :align: center
+   :alt: The down payment invoices page in Odoo Sales.
+
+To complete the flow, select the invoice with the :guilabel:`Not Paid` status, which opens that
+invoice. On the invoice page, click :guilabel:`Confirm`, then :guilabel:`Register Payment`, and
+confirm that the remaining :guilabel:`Amount` is correct. Then, click :guilabel:`Create Payment`.
+
+That pop-up disappears, and a green `Paid` banner is now present on the invoice. The line items on
+the invoice show that the total amount has been paid, including mentions of the down payment, and
+the flow is complete.
+
+.. image:: down_payment/invoice-complete-paid-banner-2.png
```

Comment:
similar here to the first one...
can crop just to show the banner (in which case we might not need), or go up to 933px to show the whole form, being sure to crop away anything that's not necessary on the UI

Reply:
Since the banner is the same as above, I believe the cropped image of the banner is redundant in this case - so I'm going to go ahead and delete the image altogether.